app/controllers/api/base_controller/renderer.rb

```diff
@@ -252,11 +252,11 @@ def split_virtual_attribute(attr)
       end
 
       def attr_accessible?(object, attr)
-        return false unless object && object.respond_to?(attr)
-        object.class.has_attribute?(attr) ||
-          object.class.reflect_on_association(attr) ||
-          object.class.virtual_attribute?(attr) ||
-          object.class.virtual_reflection?(attr)
+        return true if object && object.respond_to?(attr)
```

Comment:
do you really want `respond_to`?
That means it is anything. Even a non "attribute"

you may want to check if it is an alias.